patches/tModLoader/Terraria.ModLoader.IO/TagCompound.cs

```diff
@@ -73,6 +81,11 @@ public bool GetBool(string key)
 			return o as double? ?? o as float? ?? 0;
 		}
 
+		public T GetSerializable<T>(string key) where T : TagSerializable
+		{
+			return TagSerializables.Deserialize<T>(GetCompound(key));
+		}
```

Comment:
Yeah, I kinda forgot to do so myself. I can do it after work today, unless you want to do so yourself.